packages/contracts/contracts/voting/majority/MajorityVotingBase.sol

```diff
@@ -447,33 +483,18 @@ abstract contract MajorityVotingBase is
             !proposal_.executed;
     }
 
-    /// @notice Calculates the relative of a value with respect to a total as a percentage.
-    /// @param _value The value.
-    /// @param _total The total.
-    /// @return returns The relative value as a percentage.
-    function _calculatePct(uint256 _value, uint256 _total) internal pure virtual returns (uint256) {
-        if (_total == 0) {
-            revert ZeroValueNotAllowed();
-        }
-
-        return (_value * PCT_BASE) / _total;
-    }
-
     /// @notice Internal function to update the plugin-wide proposal vote settings.
     /// @param _votingSettings The voting settings to be validated and updated.
     function _updateVotingSettings(VotingSettings calldata _votingSettings) internal virtual {
-        if (_votingSettings.supportThreshold > PCT_BASE) {
-            revert PercentageExceeds100({
-                limit: PCT_BASE,
+        if (_votingSettings.supportThreshold > RATIO_BASE - 1) {
+            revert RatioOutOfBounds({
+                limit: RATIO_BASE - 1,
```

Comment:
Why `- 1`?  Shouldn't we allow a unanimity threshold?

Reply:
See explanation here: https://discord.com/channels/672466989217873929/1064843552376508438/1065958566843912223

This results from doing `>` for `supportThreshold`. If we always check with `>`, a value `>100%` can never be reached.